src/OWML.Common/Interfaces/IModStorage.cs

```diff
@@ -1,10 +1,14 @@
 ﻿using Newtonsoft.Json;
+using System;
 
 namespace OWML.Common
 {
 	public interface IModStorage
 	{
-		T Load<T>(string filename, bool fixBackslashes = true, JsonSerializerSettings settings = null);
+		[Obsolete]
```

Comment:
Why obsolete? And if obsolete, why no message?

Reply:
If it's because fix slashes defaults to false, then better add a message about that

Reply:
removed obsolete attribute